flexran/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# ocp admin node
+k8shost1=perf176a.perf.lab.eng.bos.redhat.com
```

Comment:
To be consistent with other workloads (trafficgen) lets obfuscate these hostnames:

```
$ git grep redhat
trafficgen/run.sh:remhost1=perfxxx.perf.lab.eng.bos.redhat.com
trafficgen/run.sh:k8shost1=perfxxx.perf.lab.eng.bos.redhat.com
```

We probably should not even include anything under redhat.com anywhere...

Reply:
OK. I will make this change.